src/components/Buttons/CalendarButton/styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const StyledCalendarButton = styled.button`
+	position: absolute;
+	top: 5px;
+	right: 5px;
+	padding: 10px;
+	border: 4px solid blue;
+`;
+
+export default StyledCalendarButton;
```

Comment:
You should repeat yourself in the HomeButton, but put both buttons into one file and use conditional styling since the only difference is the border color.

Reply:
these are just dummy buttons, I'm going to replace them with real icons in an upcoming story.